### PR TITLE
Accept invite server

### PR DESCRIPTION
**Pull request by colevoss**
This adds the `/accept-invite` api endpoint.

### CODE REVIEW

File: server/init/initAuth/Routes/acceptInvite.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  await PendingUserService.deletePendingUser(context, pendingUser._id);
+
+  res.json({
+    authenticated: true,
+    token: generateToken(
+      userWithGroups.email,
+      userWithGroups._id.toHexString(),
+    ),
+  });
+};
```
Comment (blainekasten) [inline server/init/initAuth/Routes/acceptInvite.js]: I love the cleanliness and readablility in this function. the class+method names make it so obvious to understand what steps are happening here. Well done!

File: server/services/EmailService/sendAddedToGroupNoticeEmail.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  process.env.NODE_ENV === 'production'
+    ? 'https://www.adiohub.com'
+    : 'http://localhost:3000';
+
+const sendAddedToGroupNoticeEmail = (inviter, user, group) => {
+  const fromEmail = new helper.Email('voss.cole@gmail.com');
+  const userEmail = new helper.Email(user.email);
+
+  const subject = 'You have been added to a new Adio group.';
+  const content = new helper.Content('text/html', 'bleh');
```
Comment (blainekasten) [inline server/services/EmailService/sendAddedToGroupNoticeEmail.js]: is this necassary?

File: server/services/PendingUserService.js [hunk 3]
```diff
@@ -55,6 +74,14 @@ class PendingUserService {
...
     return deletedCount === 1;
   }
+
+  // static async acceptInvite(context, token) {
+  //   const pendingUser = await this.getPendingUserByToken(context, token);
+
+  //   if (!pendingUser) {
+  //     throw new Error(`There are no pending users with the token ${token}`);
+  //   }
+  // }
```
Comment (blainekasten) [inline server/services/PendingUserService.js]: rm this?

File: src/Signin/SignUp/index.js [hunk 4]
```diff
@@ -120,6 +121,7 @@ class SignUp extends Component {
   }
 
   render() {
+    console.log(this.props);
```
Comment (blainekasten) [inline src/Signin/SignUp/index.js]: rm this

File: src/utils/Auth/index.js [hunk 5]
```diff
@@ -1,6 +1,7 @@
 // @flow
 import type { SignupUser, UpdateStateSignature } from './types';
 import LOCAL from '../../shared/LOCAL';
+// console.log(LOCAL);
```
Comment (blainekasten) [inline src/utils/Auth/index.js]: rm this

File: src/utils/Auth/index.js [hunk 6]
```diff
@@ -63,14 +64,12 @@ const logout = () => {
   updateState({ authenticated: false });
   Object.keys(LOCAL).forEach(key => {
     localStorage.removeItem(key);
-  })
+  });
 };
 
-const signUp = async (user: SignupUser) => {
+const signUp = async (user: SignupUser, token) => {
```
Comment (blainekasten) [inline src/utils/Auth/index.js]: rm this change? token is part of `user`

File: server/init/initAuth/Routes/acceptInvite.js (hunk 1 above)
Comment (colevoss) [reply server/init/initAuth/Routes/acceptInvite.js]: 👍 Thanks

File: server/services/EmailService/sendAddedToGroupNoticeEmail.js (hunk 2 above)
Comment (colevoss) [reply server/services/EmailService/sendAddedToGroupNoticeEmail.js]: I'm not sure. I haven't tested it without sending the `content` arg

File: src/utils/Auth/index.js (hunk 6 above)
Comment (colevoss) [reply src/utils/Auth/index.js]: Oh yeah. I forgot about that.

**Pull request merged**